app/ORM/Query.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\ORM;
+
+use App\Country\Entity\CountryEntity;
+
+class Query
+{
+    /** @var string */
+    private $compareTo;
+
+    /**
+     * @param string $property
+     * @param string $comparision
+     * @param string $compareTo
+     *
+     * @return Query
+     */
+    public function where(string $property, string $comparision, string $compareTo): Query
+    {
+        $this->compareTo = $compareTo;
+
+        return $this;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function get()
+    {
+        return $this->execute();
```

Comment:
Consider using a real database or ORM instead of hardcoding the return values. This will make the application more flexible and maintainable. [important]